packages/date-picker/src/FilterDateRangePicker/components/DateRangeValidationMessage/DateRangeValidationMessage.tsx

```diff
@@ -0,0 +1,76 @@
+import React from "react"
+import { FieldMessage } from "@kaizen/draft-form"
+import {
+  DateRangeValidationMessageType,
+  DateRangeValidationStatus,
+  isDateRangeMessageType,
+} from "../../../types"
+import styles from "./DateRangeValidationMessage.module.scss"
+
+export type DateRangeValidationMessageProps = {
+  /**
+   * Updates the styling of the validation FieldMessage
+   */
+  status?: DateRangeValidationStatus
+  /**
+   * A descriptive message for `status` states
+   */
+  validationMessage?: DateRangeValidationMessageType
+  errorMessageId?: string
+  isReversed?: boolean
+}
+
+export const DateRangeValidationMessage = ({
+  status,
+  validationMessage,
+  errorMessageId,
+  isReversed,
+}: DateRangeValidationMessageProps): JSX.Element | null => {
+  if (!isDateRangeMessageType(validationMessage)) {
+    return (
+      <FieldMessage
+        id={errorMessageId}
+        message={validationMessage}
+        status={status?.dateStart || status?.dateEnd}
+        reversed={isReversed}
+        classNameOverride={styles.fieldMessageIcon}
+      />
+    )
+  } else if (status?.dateEnd === status?.dateStart) {
+    return (
```

Comment:
```suggestion
  } 
  
  if (status?.dateEnd === status?.dateStart) {
    return (
```

When you have early returns, you don't need to use `else` :)